TARGETS

```diff
@@ -169,36 +169,36 @@ cpp_library(
         "options/options_sanity_check.cc",
         "port/port_posix.cc",
         "port/stack_trace.cc",
-        "table/adaptive_table_factory.cc",
-        "table/block.cc",
-        "table/block_based_filter_block.cc",
-        "table/block_based_table_builder.cc",
-        "table/block_based_table_factory.cc",
-        "table/block_based_table_reader.cc",
-        "table/block_builder.cc",
+        "table/adaptive/adaptive_table_factory.cc",
+        "table/blockbased/block.cc",
+        "table/blockbased/block_based_filter_block.cc",
+        "table/blockbased/block_based_table_builder.cc",
+        "table/blockbased/block_based_table_factory.cc",
+        "table/blockbased/block_based_table_reader.cc",
+        "table/blockbased/block_builder.cc",
+        "table/blockbased/block_prefix_index.cc",
+        "table/blockbased/data_block_hash_index.cc",
+        "table/blockbased/data_block_footer.cc",
+        "table/blockbased/flush_block_policy.cc",
+        "table/blockbased/full_filter_block.cc",
+        "table/blockbased/index_builder.cc",
+        "table/blockbased/partitioned_filter_block.cc",
```

Comment:
I don't have opinion on "blockbased" or "block_based" by itself, but since we already call "block_based" in file name, can we keep it consistent?